IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Program.cs

```diff
@@ -13,69 +13,71 @@
 
 namespace ${Namespace}
 {
-	#if __IOS__ || __TVOS__
-	[Register("AppDelegate")]
-	class Program : UIApplicationDelegate
-	#else
-	static class Program
-	#endif
+    #if __IOS__ || __TVOS__
+    [Register("AppDelegate")]
+    class Program : UIApplicationDelegate
+    #else
+    static class Program
+    #endif
     {
         private static Game1 game;
 
-		internal static void RunGame() 
-		{
-			game = new Game1();
-			game.Run();
-		}
-			
+        internal static void RunGame() 
+        {
+            using (game = new Game1())
```

Comment:
might as well remove the static and replace it with var if we are going to use a using